src/index.ts

```diff
@@ -307,6 +307,12 @@ export interface PolicyDocument {
   url: string
 }
 
+export enum SwitchTextRenderLogic {
+  SWITCH_TEXT_RENDER_ALL = 1,
+  SWITCH_TEXT_RENDER_LEGAL_BASIS = 2,
```

Comment:
can we add a couple comments what these mean. i don't get it

Reply:
Yeahhh fair

Reply:
Done